cookbook/deployment/cluster/notifications.py

```diff
@@ -1,8 +1,118 @@
 """
 Notifications
---------------------------------------------
+-------------
 
-.. NOTE::
+When a workflow completes, users can be notified by:
 
-    Coming soon 🛠
-"""
\ No newline at end of file
+* email
+* `pagerduty <https://www.pagerduty.com/>`__
+* `slack <https://slack.com/>`__
+
+The content of these notifications is configurable at the platform level.
+
+*****
+Usage
+*****
+
+When a workflow reaches a specified `terminal workflow execution phase <https://github.com/flyteorg/flytekit/blob/v0.16.0b7/flytekit/core/notification.py#L10,L15>`__
+the :py:class:`flytekit:flytekit.Email`, :py:class:`flytekit:flytekit.PagerDuty`, or :py:class:`flytekit:flytekit.Slack`
+objects can be used in the construction of a :py:class:`flytekit:flytekit.LaunchPlan`.
+
+For example
+
+.. code:: python
+
+    from flytekit import Email, LaunchPlan
+    from flytekit.models.core.execution import WorkflowExecutionPhase
+
+    # This launch plan triggers email notifications when the workflow execution it triggered reaches the phase `SUCCEEDED`.
+    my_notifiying_lp = LaunchPlan.create(
+        "my_notifiying_lp",
+        my_workflow_definition,
+        default_inputs={"a": 4},
+        notifications=[
+            Email(
+                phases=[WorkflowExecutionPhase.SUCCEEDED],
+                recipients_email=["admin@example.com"],
+            )
+        ],
+    )
+
+
+See detailed usage examples in the :std:ref:`User Guide <cookbook:sphx_glr_auto_deployment_workflow_lp_notifications.py>`
+
+Notifications can be combined with schedules to automatically alert you when a scheduled job succeeds or fails.
+
+Future work
+===========
```

Comment:
Is this still relevant? Or has the work been done?

Reply:
@wild-endeavor ?

Reply:
This is the same thing as this comment right? https://github.com/flyteorg/flytesnacks/pull/212/files#diff-6d390c83c16c1a69baf14ea8232b9acc468ebbf270ffafb2bce227ae6543ebd2R108

cc @tnsetting

currently raw events are getting pushed, so that part is done, but there's additional work here: https://github.com/flyteorg/flyteidl/pull/161 that still needs to be merged and admin updated to allow for more meaningful event structure.

Reply:
Got it. Thanks.